README.md

```
@@ -14,26 +14,24 @@ Este repositório tem o intuito de listar o nome de empresas que trabalham com P
-| [Apiki](https://apiki.com)                                                 |                                                                        | [Clique aqui](https://apiki.solides.jobs)                                               |
+| [aiqfome (Grupo Magalu)](https://aiqfome.com)                                             |                                                                        | [Clique aqui](https://magazineluiza.inhire.app/vagas)                                                  |
+| [Apiki](https://apiki.com)                                                 |                                                                        | [Clique aqui](https://carreira.apiki.com/)                                               |
 | [Bartofil](https://www.bartofil.com.br)                                    |                                                                        | [Clique aqui](https://bartofil.empregare.com/pt-br/vagas)                               |
 | [Bio Extratus](https://www.bioextratus.com.br)                             |                                                                        | [Clique aqui](https://trabalheconosco.bioextratus.com.br)                               |
 | [Bueno Networks](https://buenonetworks.com.br)                             |                                                                        | [Clique aqui](https://www.linkedin.com/company/bueno-networks/jobs)                     |
 | [Comprovei by Nstech](https://comprovei.com)                               |                                                                        | [Clique aqui](https://www.linkedin.com/company/comprovei/jobs)                          |
-| [Comunicare Aparelhos Auditivos](https://comunicareaparelhosauditivos.com) |                                                                        | [Clique Aqui](https://www.linkedin.com/company/comunicare-aparelhos-auditivos/jobs)     |
-| [Convenia](http://convenia.com.br)                                         |                                                                        | [Clique aqui](https://convenia-tech.gupy.io)                                            |
-| [Creditas](https://www.creditas.com)                                       |                                                                        | [Clique aqui](https://careers.creditas.com)                                             |
-| [Croct](https://croct.com)                                                 |                                                                        | [Clique aqui](https://boards.greenhouse.io/creditas)                                    |
+| [Comunicare Aparelhos Auditivos](https://comunicareaparelhosauditivos.com) |                                                                        | [Clique Aqui](https://www.linkedin.com/company/comunicare-aparelhos-auditivos/jobs)     |                                      |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)